samples/drivers/ipm/ipm_mcux/sysbuild.cmake

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2022 NXP
+
+if("${BOARD}" STREQUAL "lpcxpresso54114_m4")
+  set(BOARD_REMOTE "lpcxpresso54114_m0")
+elseif("${BOARD}" STREQUAL "lpcxpresso55s69_cpu0")
+  set(BOARD_REMOTE "lpcxpresso55s69_cpu1")
+else()
+  message(FATAL_ERROR "${BOARD} was not supported for this sample")
+endif()
```

Comment:
I believe this should also be updated to use the Kconfig approach.

That will allow downstream users to easily extend the Kconfig setting with downstream only remote board without patching Zephyr repo.